### ISSUE TITLE
issues related to initDatabase

### ISSUE THREAD

**Issue by srozb** [sergiotapia/torrentinim#23]
Hey, 
I've cloned master branch and built sources with `nimble build` (using nim 1.4.8 on MacOS). After successful build I am unable to initialize sqlite database as below:

`NUKE_MY_DATABASE=true ./torrentinim`

```                                                                                                                                                           
[system] Database nuke requested. Clearing all database tables and data.
/Users/slawek/src-oth/torrentinim/src/torrentinim.nim(23) torrentinim
/Users/slawek/src-oth/torrentinim/src/database.nim(18) initDatabase
/usr/local/Cellar/nim/1.4.8/nim/lib/impure/db_sqlite.nim(227) exec
/usr/local/Cellar/nim/1.4.8/nim/lib/impure/db_sqlite.nim(147) dbError
Error: unhandled exception: near "-": syntax error [DbError]
```

I believe the problem persist because of lack of quotes around `utf-8` [here](https://github.com/sergiotapia/torrentinim/blob/master/src/database.nim#L18).

**Comment by srozb** [comment]
Fix merged in #24.

There is however remaining issue related to database not being properly initialized which results in torrents pretending to be inserted into the DB while in fact they aren't.

I think [here's](https://github.com/sergiotapia/torrentinim/blob/master/src/crawlers/eztv.nim#L34) a line to blame and this applies to other crawlers too.

According to [db_sqlite documentation](https://nim-lang.org/docs/db_sqlite.html#tryInsertID%2CDbConn%2CSqlQuery%2Cvarargs%5Bstring%2C%5D), `proc tryInsertID()` will return `-1` in case of error and aforementioned line just discarding this fact with no warning/error.

I'd suggest to change application logic (perhaps `proc insert_torrent()`?) so user is notified of errors related to db operations.

**Comment by sergiotapia** [comment]
Thanks for the report @srozb! I changed the code so you aren't told an insert happened when it didn't. Much better!
